roles/redpanda_broker/templates/configs/defaults.j2

```diff
@@ -1,55 +1,78 @@
-cluster:
```

Comment:
It's not essential for this PR and I suppose I could always pull it out. That said I kind of consider this another major version jump either way so I'm not super concerned about backwards compatibility.

Reply:
I'd prefer we pull it out. Moving to directly managing the repo configs should still result in no real behavioral change to the user. It's implementation detail more than anything.

If I understand the json thing, the user would have to go change their ansible if they happen to be using their own additions to custom_config_template.

Reply:
Well TIL custom_config_template and custom_config_template**s** are not the same thing: our templates. I'm going to double check that I can just leave that bit yaml

Reply:
@hcoyote ok, so on review -- as opposed to hot take mode -- custom_config_template is just the loop_var for custom_config_templates. That means users can't* define a new value for it because it is scoped to the loop. 

Users are welcome to replace custom_config_templates, but that's an unsupported path. If they do it, they should fork the repo and carefully decide what they want to include.

The intended method for on the fly modification of config is passing JSON in through the redpanda variable. 

* I mean in theory they can't, but in practice it's python so 🤷‍♀️

Reply:
I meant custom_config_templates, sorry!